src/api/mod.rs

```diff
@@ -49,7 +49,36 @@ fn config_json_res(res: &mut Response) {
     res.set(MediaType::Json);
     res.set(StatusCode::Ok);
 }
+/*
+fn handle_db<'a>(req: &mut Request, mut res: Response<'a>) -> MiddlewareResult<'a> {
```

Comment:
well, you did a good job with nickel -- what you did is pretty much correct (not sure it is *performant* but we can worry about that later 😄).

I was just wondering what this was for. You can probably safely delete it.